microsoft-onedrive/1.0/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,54 @@
+= Microsoft OneDrive Connector 1.0 - Mule 4
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
+
+Anypoint Connector for Microsoft OneDrive (OneDrive Connector) enables you to access Microsoft Graph APIs to provide a unified programmability model that you can use to access the tremendous amount of data in Microsoft 365, Windows 10, and Enterprise Mobility + Security.
+
+For compatibility information and fixed issues, see the Microsoft OneDrive Connector Release Notes.
+
+== Prerequisites
+
+To use this connector, you must be familiar with:
+
+* Microsoft Graph API 1.0
+* Anypoint Connectors
+* Mule runtime engine (Mule)
+* Elements and global elements in a Mule flow
+* Creating a Mule app using Anypoint Studio (Studio)
+
+Before creating an app, you must have:
+
+* Credentials to access the target resource
+* Anypoint Studio version 7.3.5 or later
+
+== Common Use Cases for the Connector
+
+Some common use cases for OneDrive Connector include:
+
+* Sales Operations
+** When a sales contract comes in through the Salesforce CRM system, upload the contract to OneDrive and collaboratively review the contract with the Deal Desk. You can assign approval tasks, which when approved or rejected, notifies the contract owner.
+** When a new budget proposal or customer invoice is generated, upload the documents to OneDrive and collaborate with the legal team for their review. Assign tasks, which when approved or rejected, notifies the invoice owner.
+** When an opportunity is closed (won), create a folder structure in OneDrive based on a predefined template.
+* Employee onboarding and offboarding 
+** When a new employee starts their job, existing resources can be shared with them via OneDrive. 
+** Create a new folder in OneDrive and share it with the new employee so they can upload documents to it. 
+** When a new document is uploaded to this folder, HR is notified.
```

Comment:
This implies that OneDrive Connector has triggers.
This is not the case with 1.0.0 release.

Reply:
@dancazacu1 Are you talking about line 35 only? I assume so. Just want to confirm.

Reply:
yes